spec/classes/azure_spec.rb

```diff
@@ -0,0 +1,206 @@
+require 'spec_helper'
+
+describe 'azure' do
```

Comment:
No, you are grand! Just the : describe 'Need a descriptive tag here about the test' :)